### PR TITLE
Update android-button-example.md

### PR DESCRIPTION
**Pull request by leticiarossi**
Some comments/questions:

1. For the accessibility section, I'm not sure it needs to exist. There isn't anything in specific that the user needs to do to make buttons accessible (they should already be), unless I'm missing something. Buttons extend TextView so its text content is automatically provided to accessibility services.

2. Should we start with the Contained Button instead of Text Button since it's the default type?

3. I remember we discussed that we'd possibly have some type of notes section. The note that "<Button> is auto-inflated as (...)" could be included in the beginning so it's not super repetitive across the file?

4. I think it's a little weird to include CheckBox (a different component that has its own documentation) as a button type. On material.io it says that "Icons can be used as toggle buttons" which just seems like a high level concept. Maybe just saying that CheckBoxes can be used for that use case in the image and then linking to the CheckBox documentation would be a better approach?

#### Don't forget:
- [ ] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [ ] Sign the CLA bot. You can do this once the pull request is submitted.

[Contributing](./contributing/README.md#pull-requests) has more information and tips for a great
pull request.

### CODE REVIEW

File: button-examples/Android/android-button-example.md [hunk 1]
```diff
@@ -215,9 +208,9 @@ An outlined button has a text label, a stroked container and an optional icon.
 
 ![Outlined button anatomy diagram](assets/outlined_button_anatomy.png)
 
-1. Text label
-1. Container
-1. Icon
+* A. Text label
```
Comment (mingjane-work) [inline button-examples/Android/android-button-example.md]: We're going to eventually replace the letter callouts with numbers.

**Comment by mingjane-work** [comment]
> Some comments/questions:
> 
> 1. For the accessibility section, I'm not sure it needs to exist. There isn't anything in specific that the user needs to do to make buttons accessible (they should already be), unless I'm missing something. Buttons extend TextView so its text content is automatically provided to accessibility services.

In that case, we should call out that the button text is OCR-readable and link to TextView.

> 2. Should we start with the Contained Button instead of Text Button since it's the default type?

We're following the schema outlined in the design doc, so even if Contained is the Android default, it's still listed third. It might be worth mentioning that it's the default.

> 3. I remember we discussed that we'd possibly have some type of notes section. The note that " is auto-inflated as (...)" could be included in the beginning so it's not super repetitive across the file?

It looks these notes are part of theming. Can we move it to the bottom with the theming section?

> 4. I think it's a little weird to include CheckBox (a different component that has its own documentation) as a button type. On material.io it says that "Icons can be used as toggle buttons" which just seems like a high level concept. Maybe just saying that CheckBoxes can be used for that use case in the image and then linking to the CheckBox documentation would be a better approach?

We can also mention CheckBoxes as an alternative to Icons.

As for the inclusion in the buttons dev usage articles, we're using the usage articles as a bridge from design to developer docs because the design articles aren't necessarily linking to more than one platform developer page. Until designers decide to make any changes we should use the design article definitions of a button, and that includes lumping a code example for an icon button or checkbox with the toggle button.

File: button-examples/Android/android-button-example.md (hunk 1 above)
Comment (leticiarossi) [reply button-examples/Android/android-button-example.md]: got it! will revert it.

Comment (mingjane-work) [reply button-examples/Android/android-button-example.md]: FYI, anatomy diagrams updated in my branch.

**Comment by leticiarossi** [comment]
Changed the PR to just be adding the note about the contained button.

It's ready to be merged if you think it looks ok!

**Pull request merged**